q2_diversity_lib/tests/test_beta.py

```diff
@@ -321,10 +322,152 @@ def test_accepted_types_have_consistent_behavior(self):
         rel_freq_table = self.rf_table_as_BIOMV210Format
         accepted_tables = [freq_table, rel_freq_table]
         for table in accepted_tables:
-            actual = weighted_unifrac(table=table,
-                                      phylogeny=self.tree_as_NewickFormat)
+            actual = weighted_unifrac(
+                table=table, phylogeny=self.tree_as_NewickFormat)
             self.assertEqual(actual.ids, self.expected.ids)
             for id1 in actual.ids:
                 for id2 in actual.ids:
                     npt.assert_almost_equal(actual[id1, id2],
                                             self.expected[id1, id2])
+
+
+class BetaPassthroughTests(TestPluginBase):
+    package = 'q2_diversity_lib.tests'
+
+    def setUp(self):
+        super().setUp()
+        self.method = self.plugin.actions['beta_passthrough']
+        self.available_metrics = beta.METRICS['NONPHYLO']['UNIMPL']
+        empty_table = biom.Table(np.array([]), [], [])
+        self.empty_table = Artifact.import_data('FeatureTable[Frequency]',
+                                                empty_table)
+        crawford_tbl = self.get_data_path('crawford.biom')
+        self.crawford_tbl = Artifact.import_data('FeatureTable[Frequency]',
+                                                 crawford_tbl)
+        table = biom.Table(np.array([[0, 1, 3], [1, 1, 2]]),
+                           ['O1', 'O2'],
+                           ['S1', 'S2', 'S3'])
+        self.table = Artifact.import_data('FeatureTable[Frequency]', table)
+
+    def testMethod(self):
```

Comment:
Should this be `test_method`?

```suggestion
    def test_method(self):
```